tools/download.sh

```diff
@@ -25,21 +33,22 @@ case "$OSTYPE" in
                     echo "curl not found, installing..."
                     sudo yum install -y curl
                 fi
-
-                package=powershell-6.0.0_alpha.18-1.el7.centos.x86_64.rpm
+                get_currentpshversion
+                package=powershell-$release-1.el7.centos.x86_64.rpm
                 ;;
             ubuntu)
                 if ! hash curl 2>/dev/null; then
                     echo "curl not found, installing..."
                     sudo apt-get install -y curl
                 fi
+                get_currentpshversion
 
                 case "$VERSION_ID" in
                     14.04)
-                        package=powershell_6.0.0-alpha.18-1ubuntu1.14.04.1_amd64.deb
```

Comment:
We're on the same page!  I am thinking that when I add the repos I should make the initial install request *specifying* the version as the version that comes down in the version tag on git.  That way if there happens to a sync issue where a newer than the latest "official" release is hanging around, this code only installs the official release tagged as latest on git?